src/test/java/integration/CounterpartyControllerIT.java

```diff
@@ -78,6 +84,43 @@ public void getCounterparty() throws Exception {
                 statusCode(SC_OK).
                 body("uuid", equalTo(counterpartyUuid.toString()));
     }
+    
+    @Test
+    public void getAllClientsOfCounterparty() {
```

Comment:
Plz, rename to getClientsByCounterpartyUuid as method in CounterpartyController name
Plz, see other new methods in the file